docs/expedient.elastic.ece_snapshot_repo_module.rst

```diff
@@ -8,7 +8,7 @@ expedient.elastic.ece_snapshot_repo
 **Create or delete a snapshot repository**
 
 
-Version added: 2.9
+Version added: 1.0.0
```

Comment:
am I understanding "version added" wrong? I thought this was to reference an ansible version?

Reply:
ok yea I misunderstood that field, thanks for clarifying